ansible/roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -18,6 +18,16 @@
     mode: 0640
   tags: [ini]
 
+- name: Set owner on grafana.ini
+  file:
+    path: /etc/grafana
+    state: directory
+    # This is the UID used by the grafana/grafana container
+    owner: 104
```

Comment:
This would not always work, if the base image changed (or we used a different base image), the UID would be different. We should patch the grafana entrypoint to chown /etc/grafana/grafana.ini instead. That is if we actually want to use the config. Everything in the config is configurable with cli options when starting grafana and imo, it is better to use these options instead.

Reply:
ok. I do want to make sure this works outside of the RHEL context though.

What would you chown to in the downstream container?

Reply:
In the container, you can chown directly to the grafana user, you don't need to use the uids. That is what they are currently doing for /var/lib/grafana anyway.